preferences.py

```diff
@@ -154,15 +154,16 @@ class LOOKINGGLASS_PT_preferences(AddonPreferences):
 									name="Logging Mode",
 									update=LookingGlassAddon.update_logger_levels,
 									)
+	console_output: bpy.props.BoolProperty(
+									default=False,
+									name="Log to console",
+									description="Additionally log outputs to std out for debugging",
+									update=LookingGlassAddon.update_console_output,
+									)
+
 	# draw function
 	def draw(self, context):
-
 		layout = self.layout
-
-		# render mode
-		row_render_mode = layout.row()
-		row_render_mode.prop(self, "render_mode")
-
-		# logger level
-		row_logger_level = layout.row()
-		row_logger_level.prop(self, "logger_level")
+		layout.prop(self, "render_mode")
+		layout.prop(self, "logger_level")
+		layout.prop(self, "console_output")
```

Comment:
Thanks for adding this as well! I just would do some optical changes for the UI.
```suggestion
# render mode
row_render_mode = layout.row()
column_1 = row_render_mode.column()
column_1.label(text="Render Mode:")
column_1.scale_x = 0.2
column_2 = row_render_mode.column()
column_2.prop(self, "render_mode", text="")
column_2.scale_x = 0.8

# logger level
row_logger = layout.row()
column_1 = row_logger.column()
column_1.label(text="Logging Mode:")
column_1.scale_x = 0.2
column_2 = row_logger.column()
column_2.prop(self, "logger_level", text="")
column_2.scale_x = 0.55
column_3 = row_logger.column()
column_3.prop(self, "console_output")
column_3.scale_x = 0.25
```